initfiles/componentfiles/configxml/@temp/esp_service_WsSMC.xsl

```diff
@@ -125,6 +125,46 @@ This is required by its binding with ESP service '<xsl:value-of select="$espServ
             <xsl:with-param name="bindingNode" select="$bindingNode"/>
             <xsl:with-param name="authNode" select="$authNode"/>
         </xsl:apply-templates>
+        <xsl:apply-templates select="." mode="ws_store">
+            <xsl:with-param name="bindingNode" select="$bindingNode"/>
+            <xsl:with-param name="authNode" select="$authNode"/>
+        </xsl:apply-templates>
+    </xsl:template>
+
+<!-- WS-Store -->
+    <xsl:template match="EspService" mode="ws_store">
+        <xsl:param name="bindingNode"/>
+        <xsl:param name="authNode"/>
+
+        <xsl:variable name="serviceType" select="'ws_store'"/>
+        <xsl:variable name="serviceName" select="concat($serviceType, '_', @name, '_', $process)"/>
+        <xsl:variable name="bindName" select="concat($serviceType, '_', $bindingNode/@name, '_', $process)"/>
+        <xsl:variable name="bindType" select="'ws_storeSoapBinding'"/>
+        <xsl:variable name="servicePlugin">
+            <xsl:call-template name="defineServicePlugin">
+                <xsl:with-param name="plugin" select="'ws_store'"/>
+            </xsl:call-template>
+        </xsl:variable>
+        <EspService name="{$serviceName}" type="{$serviceType}" plugin="{$servicePlugin}">
+            <xsl:element name="StoreProvider">
+                <xsl:attribute name="lib">espstorelib</xsl:attribute>
+            </xsl:element>
+            <xsl:element name="Stores">
+                <xsl:element name="Store">
+                    <xsl:attribute name="description">ECL Watch User Preferences Store</xsl:attribute>
+                    <xsl:attribute name="name">ECLWatchPreferences</xsl:attribute>
```

Comment:
@GordonSmith I'm adding wsstore to wssmc by default and creating a top level store (and setting it as the default store). Let me know if the name "ECLWatchPreferences" is compatible w/ your intent for this store

Reply:
@rpastrana HPCCClientPreferences would work better?

Reply:
@GordonSmith sure...